js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.js

```diff
@@ -55,170 +71,384 @@ const PreLaunchChecklist = ( props ) => {
 				<Section.Card>
 					<Section.Card.Body>
 						<VerticalGapLayout size="large">
-							<div className="gla-pre-launch-checklist__checkbox_popover">
+							{ ! values.website_live && (
+								<div className="gla-pre-launch-checklist__checkbox">
+									<CheckboxControl
+										{ ...getInputProps( 'website_live' ) }
+									></CheckboxControl>
+									<FlexBlock>
+										<Panel>
+											<PanelBody
+												title={ __(
+													'My store is live and accessible to all users',
+													'google-listings-and-ads'
+												) }
+												initialOpen={ true }
```

Comment:
Didn't add the figma link here. Sorry about that. Just added it in the description. This state should be true here. We will open the card if the check doesn't pass, otherwise it will be closed.

Reply:
> Just added it in the description.

![image](https://user-images.githubusercontent.com/17420811/189570633-bc6523ba-708f-487c-b9c0-0dec2e5d39e2.png)

Thanks for adding the document. The added Figma link points to the same document I was referring to, and the first visual design shows all the unchecked items are collapsed. And that's why I inferred their initial state might be closed.

> This state should be true here. We will open the card if the check doesn't pass, otherwise it will be closed.

Sorry, I couldn't find the relevant document/spec in Figma, could you point me where it's?

Reply:
Sorry. I don't find the doc recording this. I confirmed this with PM before. I will confirm them again and bring the doc with you soon about this.

Reply:
[Confirmed](https://docs.google.com/document/d/1dcyZN6t0AgG36TCqV29luRLzrCI9ArS6VUHDsVoU5Yo/edit#bookmark=id.ey9n5s26n5n3) this with UX team, we will keep it close initially.